pkg/k8s/pods/pod.go

```diff
@@ -177,6 +177,16 @@ func GetUserByPod(ctx context.Context, p *apiv1.Pod, container string, config *r
 	return userID, nil
 }
 
+//HasNodeModules returns if the container has node_modules
+func HasNodeModules(ctx context.Context, p *apiv1.Pod, container string, config *rest.Config, c *kubernetes.Clientset) bool {
+	cmd := []string{"sh", "-c", "[ -d 'node_modules' ] && echo 'node_modules exists'"}
+	out, err := execCommandInPod(ctx, p, container, cmd, config, c)
+	if err != nil {
+		return false
+	}
+	return strings.Contains(out, "ode_modules exists")
```

Comment:
is the `n` on `node_modules` missing o purpose? Also, could we just check the exit code instead of depending on  string?